source/setup/nao_setup.rst

```diff
@@ -54,19 +57,18 @@ Flashing the NAO
 *********************************
 Robot Config, Name and Wifi Setup
 *********************************
+If the robots are new you will still need to:
 
-Run
+* Add ``<robot-name>`` to the list of robot names in ``$RUNSWIFT_CHECKOUT_DIR/bin/source.sh``
+* Add ``<robot-name>`` and the ip address of the robot to ``utils/wifitools/updateWlanSetup.py``
+* Create a copy of the default ``.cfg`` file called ``<robot-name>.cfg`` in ``$RUNSWIFT_CHECKOUT_DIR/image/home/nao/data/configs/``
+* Create a copy of the default ``.cfg`` file called``<robot-name>.cfg`` in ``$RUNSWIFT_CHECKOUT_DIR/image/home/nao/data/configs/body/``
 
-.. code-block:: bash
+*********************************
+Syncing the nao
+*********************************
+Finally, for all robots, run
 
-    setup-robot.sh nao.local mario
+.. code-block:: bash
 
-.. note::
-    Replace ``mario`` with the name of the new robot.
-    Robot names should consist of lower case alphabets or numbers
-
-This step
-* Create ``<robot-name>.cfg`` in ``$RUNSWIFT_CHECKOUT_DIR/image/home/nao/data/configs/``
-* Create ``<robot-name>.cfg`` in ``$RUNSWIFT_CHECKOUT_DIR/image/home/nao/data/configs/body/``
-* Adds <robot-name> to list of robot names in ``$RUNSWIFT_CHECKOUT_DIR/bin/source.sh``
-* Adds <robot-name> and ip address of robot to ``utils/wifitools/updateWlanSetup.py``
+    nao_sync -s 
```

Comment:
probably needs the `-H nao.local $robot_name`

Reply:
You have my full blessing to make these changes. All this PR was trying to be was to add a one line warning - you're the one that wrote the setup stuff so this would be a far less painful experience for everyone if you made the change

Reply:
can i rewrite the whole page?

Reply:
Go for a it